pom.xml

```diff
@@ -9,111 +9,80 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <javafx.version>11</javafx.version>
+        <javafx.version>11.0.2</javafx.version>
+        <javafx.scope>provided</javafx.scope>
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
```

Comment:
oops missed these properties, but they're overridden by the plugin below anyways.